lib/feed/deleteFeedItem.ts

```diff
@@ -0,0 +1,18 @@
+import { doc, deleteDoc } from "firebase/firestore";
+
+import { feedCol } from "../firestore";
+
+import type { FeedCommentType, FeedItemType } from "@/components/feed";
+
+const deleteFeedItem = async (
+  postData: FeedItemType,
+  commentData?: FeedCommentType
```

Comment:
```suggestion
  postId: string,
  commentId?: string
```

Just pass the IDs in we don't need all of it

Reply:
Deleting a post should also delete the respective comments, otherwise firestore will have a bunch of half-complete items